src/bin/cli/mod.rs

```diff
@@ -515,27 +515,32 @@ impl PackageTrait for DocPackage {
 pub fn imports_arg<T: PackageTrait>(package: &T) -> FileResolver {
     let mut resolver = FileResolver::default();
 
-    for filename in package.get_input() {
-        if let Ok(path) = PathBuf::from(filename).canonicalize() {
-            let _ = resolver.add_import_path(path.parent().unwrap());
-        }
-    }
-
+    let mut added_path = false;
     if let Some(paths) = package.get_import_path() {
         for path in paths {
-            if let Err(e) = resolver.add_import_path(path) {
-                eprintln!("error: import path '{}': {}", path.to_string_lossy(), e);
-                exit(1);
-            }
+            resolver.add_import_path(path);
+            added_path = true;
         }
     }
+    if !added_path {
```

Comment:
This deserves a comment. Even better if the function had a Doc comment explaining it all and linking to https://docs.soliditylang.org/en/v0.8.17/path-resolution.html#base-path-and-include-paths